how-to.v2/use-a-sidecar-app-basic/platform/client/src/provider.ts

```diff
@@ -0,0 +1,217 @@
+import OpenFin from "@openfin/core";
+import type { ChannelClient } from "@openfin/core/src/OpenFin";
+
+let loggingElement: HTMLElement | null;
+let clearLogsBtn: HTMLButtonElement | null;
+let connectToSideCarBtn: HTMLButtonElement | null;
+let dispatchMessageToSideCarBtn: HTMLButtonElement | null;
+let channelClient : ChannelClient;
+
+const SIDECAR_APP_ALIAS = "sidecar-app";
+const SIDECAR_APP_TARGET = "SideCar.App.exe";
+const SIDECAR_APP_SRC = `${ location.origin }/assets/sidecar-app.zip`; 
+/**
+ * Wait for the DOM to have been loaded before we connect the UI elements and initialize the platform.
+ */
+window.addEventListener("DOMContentLoaded", async () => {
+	await initializeDom();
+	await initializePlatform();
+});
+
+/**
+ * Initialize the DOM elements.
+ */
+async function initializeDom(): Promise<void> {
+	loggingElement = document.querySelector("#logging");
+	const loggingContainer: HTMLDivElement | null = document.querySelector("#logging-container");
```

Comment:
Just a comment really if you use generic `document.querySelector<HTMLDivElement>()` the loggingContainer is already the correct type

Reply:
done